config/crd/bases/opentelemetry.io_opentelemetrycollectors.yaml

```diff
@@ -282,6 +282,7 @@ spec:
                 type: string
               tolerations:
                 description: Toleration to schedule OpenTelemetry Collector pods.
+                  This is only relevant to daemonsets, statefulsets and deployments
```

Comment:
Could you revert this change? Otherwise it might conflict with another PR that is fixing this.

Reply:
I didn't see this in other PRs. I mean even in your recent PR (you reverted it, so I was thinking to add it with a separate PR).

Reply:
This is being changed here: https://github.com/open-telemetry/opentelemetry-operator/pull/306/files#diff-c5e8337ee47f0f73be5c1b6eb6e52fff3c9cde0a18b59d696dc827e1fba75e10R285